src/Protocol/ActivityPub/Processor.php

```diff
@@ -287,7 +287,11 @@ public static function createItem($activity)
 
 		$item['uri'] = $activity['id'];
 
-		$item['created'] = DateTimeFormat::utc($activity['published']);
+		if (empty($activity['published'])) {
+			DI::logger()->notice('published key is empty for activity', ['activity' => $activity]);
```

Comment:
Possibly you could add a callstack?

Reply:
I don't see the point, this is the AP processor so the call stack will be straightforward from the AP inbox endpoints, for reference you can use the one @AlfredSK provided in the corresponding issue.

Reply:
The processor can be called when fetching posts (thread autocomplete, relay) or when receiving posts.

Reply:
And the exception this PR addresses shows that this error happens when receiving, do you need to know if it would happen when fetching posts as well?